### DISCUSSION TITLE
How may i add "alt" attribute to be returned as part of the ImageElement audit?

### DISCUSSION THREAD

**Discussion by xiaochris** [GoogleChrome/lighthouse#14854] [Q&A]
Hello, as the above title, as it seems that the `alt` attribute is not being returned or passed back as a attribute back by the ImageElement gatherer.

**Comment by connorjclark** [comment]
Are you referring to the ImageElement gatherer? https://github.com/GoogleChrome/lighthouse/blob/main/core/gather/gatherers/image-elements.js

We don't grab the `alt` of the image, but it seems like a good idea to start. Are you needing this for a custom audit?

**Reply by xiaochris** [reply]
Hi @connorjclark, yes, I am referring to the ImageElement gatherer. 

As we are creating a custom audit, therefore, there is a requirement to check for the `alt` attribute. 

not sure if there are any better way to grab the `alt` attribute instead of creating another custom gatherer to fetch the `img` tag and grab the `alt` attribute.

**Reply by paulirish** [reply]
Extending the current gatherer makes a lot of sense, yeah.

Do you want to give us a pull request?

**Reply by xiaochris** [reply]
hi @paulirish, thanks for the reply. 

just to clarify, do you want me to create a separate gatherer or do you want me to make a pull request to add in the `alt` tag into image-elements.js `getHTMLImages` function?

**Reply by paulirish** [reply]
"make a pull request to add in the alt tag into image-elements.js getHTMLImages function" <==   :)
